packages/core/src/rules/overlay1/__tests__/info-contact.test.ts

```diff
@@ -0,0 +1,74 @@
+import { outdent } from 'outdent';
+import { parseYamlToDocument, replaceSourceWithRef, makeConfig } from '../../../../__tests__/utils';
+import { lintDocument } from '../../../lint';
+import { BaseResolver } from '../../../resolve';
+
+describe('Overlay 1.0 Description', () => {
+  it('should not report if the Contact Object is valid', async () => {
+    const document = parseYamlToDocument(
+      outdent`
+        overlay: 1.0.0
+        info:
+          title: Example Overlay 1 definition. Valid.
+          version: '1.0'
+          contact:
+            name: API Support
+            url: http://www.example.com/support
+            email: support@example.com
+        extends: 'openapi.yaml'
+        actions: []
+            `,
+      'overlay.yaml'
+    );
+    const results = await lintDocument({
+      externalRefResolver: new BaseResolver(),
+      document,
+      config: await makeConfig({
+        rules: {
+          'info-contact': { severity: 'error' },
+        },
+      }),
+    });
+    expect(replaceSourceWithRef(results)).toMatchInlineSnapshot(`[]`);
+  });
+
+  it('should report if the Contact Object is not defined', async () => {
+    const document = parseYamlToDocument(
+      outdent`
+        overlay: 1.0.0
+        info:
+          title: Example Overlay 1 definition. Invalid.
+          version: '1.0'
+        extends: 'openapi.yaml'
+        actions: []
+            `,
+      'overlay.yaml'
+    );
+    const results = await lintDocument({
+      externalRefResolver: new BaseResolver(),
+      document,
+      config: await makeConfig({
+        rules: {
+          'info-contact': { severity: 'error' },
+        },
+      }),
+    });
+    expect(replaceSourceWithRef(results)).toMatchInlineSnapshot(`
```

Comment:
Could you add a test for wrong overlay structure so the content violates the `struct` rule? Or I can add it later.